src/e2e/puppeteer/__tests__/snapshot.ts

```diff
@@ -368,6 +390,28 @@ describe('drag', () => {
     const image = await screenshot()
     expect(image).toMatchImageSnapshot()
   })
+
+  it('drop hover after table', async () => {
+    await paste(`
+      - x
+      - a
+        - =view
+          - Table
+        - =pin
+          - true
+        - b
+          - c
+        - d
+          - e
+    `)
+
+    await clickThought('x')
+    await dragAndDropThought('x', 'a', { position: 'after', childrenCount: 2 })
```

Comment:
I think I would prefer an API of:

```ts
await dragAndDropThought('x', 'd', { position: 'uncle' })
```

which would mean "drag `x` after `d`, as its uncle."

`childrenCount` will not work for multiline thoughts in either col1 or col2.

Reply:
Then I think we need to find some other way to find the difference between `dragStart.y` and `dragEnd.y`.

Reply:
Sure.

Since we already have the `dropUncle` property, maybe this would make more sense:

```ts
await dragAndDropThought('x', 'd', { position: 'after', dropUncle: true })
```

Either way, the point is that the API should reflect the semantics of the caller, not the implementation.

Reply:
Right, I understand.